fern/docs/pages/recipes/list-llm.mdx

```diff
@@ -0,0 +1,28 @@
+## List of working LLM
+
+**Do you any working combination of LLM and embeddings?**
```

Comment:
Do you ___ any working. You mean "do you miss"?

Reply:
I meant "Do you have", thank you for pointing out :wink: